src/interpreter/ByteCode.h

```diff
@@ -1156,27 +1153,58 @@ struct GetObjectInlineCache {
     GetObjectInlineCacheDataVector m_cache;
 };
 
+struct GetObjectInlineCacheSimpleCase : public gc {
+    GetObjectInlineCacheSimpleCase()
+    {
+        memset(this, 0, sizeof(GetObjectInlineCacheSimpleCase));
+    }
+
+    void* operator new(size_t size);
+    void* operator new[](size_t size) = delete;
+
+    static constexpr size_t inlineBufferSize = 3;
+
```

Comment:
Is there any reason for caching only 3 items?
3 items could cover the most of `get` operations in benchmark?